README.md

```diff
@@ -386,7 +387,6 @@ myproj                        1( 12%)               1(100%)
                 ui            1(100%)               1(100%)
 ```
 
-
 Notes:
```

Comment:
As we have replaced `*Notes*` with `#### Notes`, let's do it here as well for consistency: `#### Notes`